### ISSUE TITLE
More configurability

### ISSUE THREAD

**Issue by HallerPatrick** [HallerPatrick/frosch#18]
This means for colours and maybe symbols. Frosch will in the future allow for way more configuration options for different purposes. We should therefore come up with a better config manager.

**Comment by pjz** [comment]
FWIW, I'm a fan of [everett](https://github.com/willkg/everett) ; it may be overkill.

**Comment by HallerPatrick** [comment]
Yeah for now a little overkill. I made a minimal implementation. I  am not sure if I ever want to support config files, in case this would come in handy.
